paper/paper.md

```diff
@@ -113,13 +113,23 @@ including common polymers, forcefields, and bulk system initialization algorithm
 
 # Recipes
 `flowerMD` offers the following two ready-to-go recipes to illustrate how the design creates
-potential for expanding the library of open-source and version-controlled workflows. The included example
-code demonstrates how the modularity of `flowerMD` allows use and re-use of workflows, using
- pre-built recipe subclasses included in the repository: `SlabSimulation`, `WeldSimulation`, and `Tensile`.
-This script creates two "slabs" of polyethylene, simulates welding at their interface, then simulates a tensile
-test of the resultant weld. Note that each of these steps can be run independently, and each simulation
+potential for expanding the library of open-source and version-controlled workflows. The example
+code included in this repository demonstrates how the modularity of `flowerMD` allows use and re-use of workflows, using
+pre-built "recipe steps" (subclasses) included in the repository: 
+`flowerMD.modules.welding.SlabSimulation`, `flowerMD.modules.welding.Interface`, `flowerMD.modules.welding.WeldSimulation`,
+and `flowerMD.library.simulations.Tensile`.
+
+We can use these four blocks to build the following recipe:
+1. `SlabSimulation` creates two "slabs" of e.g. polyethylene
+2. `Interface` joins two resultant slabs at an interface
+3. `WeldSimulation` simulates thermal welding at this interface, and
+4. `Tensile` simulates a tensile test of the resultant weld to create a stress/strain curve.
+Note that each of these steps can be run independently, and each simulation
 type is agnostic to system and force field selection, enabling easy iteration with different
-materials, force fields, etc. without replicating the workflow code itself.
+materials, force fields, etc. without replicating the workflow code itself. One example might be
+that we could instead run `SlabSimulation` on two different bulk polymers, then reuse the rest of
+the recipe unchanged. We could also iterate over new weld temperatures pending results of `Tensile` runs, etc.
+We hope this flexible and modular design will be a boon to many future molecular simulation researchers.
```

Comment:
Let me know how we feel about this as a closing sentence -- to cheesy?

Reply:
I like the idea of closing with a big statement, but maybe we should be more specific about it? What are some specific things we hope that package could achieve? Standardizing workflows for molecular simulations? Creates a mechanism for others to design/share their workflow designs, ....?

Reply:
IMO the ideal end goal for flower is having a bunch of fleshed out recipe workflows, but that can only happen if people decide to use flower as the starting point when making their workflow. So, we could close focusing on the idea of it being a place for people to use as a starting point.

Reply:
addressed in df2d2b6